frontend/src/components/referenceGenomes/ReferenceGenomesListContent.tsx

```diff
@@ -1,21 +1,42 @@
-import React from 'react'
-import { ReferenceGenome } from '../../models/frontend_models'
-import PageContainer from '../PageContainer'
+import React, { useEffect, useState } from 'react'
+import { getAllItems } from '../../models/frontend_models'
 import AppPageHeader from '../AppPageHeader'
 import PageContent from '../PageContent'
+import { useAppSelector } from '../../hooks'
+import { selectReferenceGenomesByID, selectTaxonsByID } from '../../selectors'
+import { Table } from 'antd'
+import AddButton from '../AddButton'
+import { getColumnsForReferenceGenome } from './ReferenceGenomeTableColumns'
+import { ReferenceGenome } from '../../models/frontend_models'
 
-export interface ReferenceGenomesListContentProps {
-	referenceGenomes: ReferenceGenome[]
-}
 
-function ReferenceGenomesListContent({referenceGenomes} : ReferenceGenomesListContentProps) {
+function ReferenceGenomesListContent() {
+	const [referenceGenomeColumns, setReferenceGenomeColumns] = useState<ReferenceGenome[]>();
```

Comment:
This is super confusing because this state is storing ReferenceGenome objects, not columns. I guess the names should be `[referenceGenomes, setReferenceGenomes]` here.